decktape.js

```diff
@@ -226,6 +218,18 @@ process.on('unhandledRejection', error => {
 });
 
 (async () => {
+  const plugins = await loadAvailablePlugins(path.join(path.dirname(fileURLToPath(import.meta.url)), 'plugins'));
```

Comment:
Loading plugins could not be done the same way dynamically without an `await`. I decided to move this inside the anonymous `async` function.